lightning/src/ln/channelmanager.rs

```diff
@@ -10505,25 +10505,31 @@ where
 	}
 
 	fn handle_message_received(&self) {
-		let invoice_requests = self.pending_outbound_payments.get_invoice_request_awaiting_invoice();
+		let mut invoice_requests = self.pending_outbound_payments.get_invoice_request_awaiting_invoice();
 		if invoice_requests.is_empty() {
 			return;
 		}
+
+		let invoice_requests_reply_paths = invoice_requests
+			.drain(..)
```

Comment:
Use `into_iter` instead

Reply:
Done! [420ed56](https://github.com/lightningdevkit/rust-lightning/commit/420ed565bf1fce14e96c02e05fa06a7b49e3a3ec)
Thanks for the suggestion!